src/Index.cmake

```diff
@@ -11,7 +11,7 @@ endif()
 include_guard()
 
 # only useable here
-set(ProjectOptions_SRC_DIR ${CMAKE_CURRENT_LIST_DIR})
+set(ProjectOptions_SRC_DIR ${CMAKE_CURRENT_LIST_DIR} CACHE INTERNAL "")
```

Comment:
This must not a `CMakeCache` variable because if more than one subproject use this ProjectOptions independently, there is a name clash!

Reply:
We should probably add a note about this. If someone uses project options in multiple submodules, it can result in errors

Reply:
The issue we're running into is that this variable is seemingly out of scope by the time a lot of project_option macros are called. The only solution I can think of would be to call the includes before macro definitions – but I'm not sure what the impact on build times would be since I guess the inclusions are delayed intentionally?

Reply:
> The issue we're running into is that this variable is seemingly out of scope by the time a lot of project_option macros are called. The only solution I can think of would be to call the includes before macro definitions – but I'm not sure what the impact on build times would be since I guess the inclusions are delayed intentionally?

you may try to change the [project_options](https://github.com/aminya/project_options)/CMakeLists.txt and append the `CMAKE_MODULE_PATH` ...